docs/mkdocs/docs/features/trailing_commas.md

```diff
@@ -0,0 +1,89 @@
+# Trailing Commas
+
+Like comments, this library does not support trailing commas in arrays and objects *by default*.
+
+You can pass set parameter `allow_trailing_commas` to `#!c true` in the parse function to allow trailing commas in arrays and objects. Trailing commas are ignored during parsing.
+
+Note that this library does not add trailing commas when serializing JSON data. Even when you set `allow_trailing_commas` to true, `{,}` and `[,]` are not valid JSON.
```

Comment:
I think this second sentence belongs as part of the previous paragraph. Something like

>  ...allow trailing commas after the last element in arrays or objects.  A single comma as the only content of the array or object (`[,]` or `{,}`)  is not allowed."

> are not valid JSON.

In any case, this is not valid JSON, but an extension, or a different file format (JWCC), so I wouldn't use this description.

I would file this whole thing under Postel's law: "Be liberal in what you accept, and conservative in what you send."